modules/st2-login/login.component.js

```diff
@@ -230,6 +231,7 @@ export default class Login extends React.Component {
               placeholder="Password"
               required
               value={this.state.password}
+              maxLength="16"
```

Comment:
I think the maxlength for password is too short in some cases. Can this be 32 as well?